### PR TITLE
copy the external_id in clone cluster

### PR DESCRIPTION
**Pull request by yongwen**
when the "clone cluster" button is clicked in the Capacity page, the external_id need to be copied over to the new stage.

### CODE REVIEW

File: deploy-board/deploy_board/webapp/cluster_view.py [hunk 1]
```diff
@@ -622,12 +622,14 @@ def clone_cluster(request, src_name, src_stage):
         ##0. teletraan service get src env buildName
         src_env = environs_helper.get_env_by_stage(request, src_name, src_stage)
         build_name = src_env.get('buildName', None)
+        external_id = src_env.get('externalId', None)
```
Comment (euccas) [inline deploy-board/deploy_board/webapp/cluster_view.py]: When external_id is None, do we need display a warning? The warning can be displayed here when cloning cluster happens, or somewhere else in the source env stage's related pages.

**Comment by edenjnbaptiste-pinterest** [comment]
Are there no tests in Teletraan?

File: deploy-board/deploy_board/webapp/cluster_view.py (hunk 1 above)
Comment (lixmgl) [reply deploy-board/deploy_board/webapp/cluster_view.py]: If external Id is None, do we still need clone it or need create it instead?

Comment (yongwen) [reply deploy-board/deploy_board/webapp/cluster_view.py]: for non-pinterest, external_id will be none. so cloning should still happen.  for pinterest, the external_id should not be none from source cluster. if so, there may already an issue from the source cluster. i feel it is better to just clone just what is from the source.

**Comment by yongwen** [comment]
> Are there no tests in Teletraan?

for UI there is no unit test.

File: deploy-board/deploy_board/webapp/cluster_view.py (hunk 1 above)
Comment (euccas) [reply deploy-board/deploy_board/webapp/cluster_view.py]: yes considering when external_id is missing we will have the "Missing project ownership info" warning, here we can just clone from source.

**Pull request merged**